### ISSUE TITLE
Optional link in notification is not available

### ISSUE THREAD

**Issue by MorrisJobke** [nextcloud/ios#779]
* having a notification with a link set (see https://github.com/nextcloud/notifications/blob/master/docs/ocs-endpoint-v2.md#notification-element for the `link` attribute)
* expected: being able to somehow access this linked resource, by clicking on the notification - that's how it works on the web and on Android (right @tobiasKaminsky?)
* actual: link is not available

cc @marinofaggiana If you wish I can send you an example notification on our Nextcloud for verification.

**Comment by MorrisJobke** [comment]
@marinofaggiana 🏓

**Comment by marinofaggiana** [comment]
Yes @MorrisJobke soon

**Comment by tobiasKaminsky** [comment]
> * on Android (right @tobiasKaminsky?)

Yes, but currently only within our app, not in push notification (outside our app)

**Comment by MorrisJobke** [comment]
> Yes, but currently only within our app, not in push notification (outside our app)

That's totally fine for now.

**Comment by marinofaggiana** [comment]
@MorrisJobke but with the touch on link it should open a new view with safari but required the insert of credential ... this is not acceptable.

**Comment by MorrisJobke** [comment]
> @MorrisJobke but with the touch on link it should open a new view with safari but required the insert of credential ... this is not acceptable.

Why not? Couldn't it open in safari instead of the view, because there you can reuse sessions.

**Comment by marinofaggiana** [comment]
@MorrisJobke yes but it's horrible ... 😱

**Comment by MorrisJobke** [comment]
Any news here?

**Comment by githubkoma** [comment]
Would love to see this as well :)
